retriever/lib/tools.py

```diff
@@ -54,44 +75,8 @@ def final_cleanup(engine):
 config_path = os.path.join(HOME_DIR, 'connections.config')
 
 
-def choose_engine(opts, choice=True):
-    """Prompts the user to select a database engine"""
-    from retriever.engines import engine_list
-
-    if "engine" in list(opts.keys()):
-        enginename = opts["engine"]
-    elif opts["command"] == "download":
-        enginename = "download"
-    else:
-        if not choice:
-            return None
-        print("Choose a database engine:")
-        for engine in engine_list:
-            if engine.abbreviation:
-                abbreviation = "(" + engine.abbreviation + ") "
-            else:
-                abbreviation = ""
-            print("    " + abbreviation + engine.name)
-        enginename = input(": ")
-    enginename = enginename.lower()
-
-    engine = Engine()
-    if not enginename:
-        engine = engine_list[0]
-    else:
-        for thisengine in engine_list:
-            if (enginename == thisengine.name.lower() or
-                    thisengine.abbreviation and
-                    enginename == thisengine.abbreviation):
-                engine = thisengine
-
-    engine.opts = opts
-    return engine
-
-
-def reset_retriever(scope):
-    """Remove stored information on scripts, data, and connections"""
-
+def reset_retriever(scope="all"):
+    """Remove stored information on scripts, data, and connections."""
```

Comment:
Expand docstring since this is user facing.

Reply:
We should have an optional argument that runs the reset automatically without checking for cases where someone wants to use this non-interactively.

Reply:
Agreed. What should the optional argument name be `force=False`?

Reply:
`force` is probably fine. `check_before_reset=True` would be a more explicit option.

Reply:
Or maybe `ask=True` or `ask_user=True`?